File: 3065-minimum-operations-to-exceed-threshold-value-i/3065-minimum-operations-to-exceed-threshold-value-i.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int count=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]<k)
            {
                count++;
            }
        }
        return count;
    }
};

File: 3065-minimum-operations-to-exceed-threshold-value-i/3065-minimum-operations-to-exceed-threshold-value-i.java
class Solution {
    public int minOperations(int[] nums, int k) {

        Arrays.sort(nums);
        int left=0;
        int res =0;

        int right=nums.length-1;
        while(left<=right){
            int mid = (left+right)/2;
            if(nums[mid]<k){
                res = res+(mid-left+1);
                left =mid+1;
            }else{
                right = mid-1;
            }
        }
        return res;
        
    }
}